docs/calicoctl/profile.md

```diff
@@ -0,0 +1,518 @@
+                    associated with the profile.
+  --at=<POSITION>   Specify the position in the chain where the rule should
+                    be placed. Default: append at end.
+
+Examples:
+  Add and set up a rule to prevent all inbound traffic except pings from the 192.168/16 subnet
+  $ calicoctl profile add only-local-pings
+  $ calicoctl profile only-local-pings rule add inbound deny icmp
+  $ calicoctl profile only-local-pings rule add inbound --at=0 allow from 192.168.0.0/16
+
+```
+
+## calicoctl profile commands
+
+
+### calicoctl profile show 
+This command prints a list of the security profiles 
+known to Calico.
+
+If the `--detailed` flag is passed into the command, 
+the command will print information about the endpoints 
+associated with each profile, including:
+ - Profile name
+ - Workload ID associated with profile
+ - Endpoint ID associated with profile
+ - Host owner of workload
+ - Orchestrator running the workload
+ - State of the workload
+
+
+This command can be run on any Calico node. 
+
+Command syntax:
+
+```
+calicoctl profile show [--detailed]
+
+    --detailed: Show information about workloads associated with profiles
+```
+
+Examples:
+
+```
+$ calicoctl profile show 
++------------+
+|    Name    |
++------------+
+| PROF_A_C_E |
+|   PROF_B   |
+|   PROF_D   |
++------------+
+
+$ calicoctl profile show --detailed
++------------+----------------+-----------------+------------------------------------------------------------------+----------------------------------+--------+
+|    Name    |      Host      | Orchestrator ID |                           Workload ID                            |           Endpoint ID            | State  |
++------------+----------------+-----------------+------------------------------------------------------------------+----------------------------------+--------+
+| PROF_A_C_E | calico-host-01 |      docker     | 15a483ab65cc60e2831859dd0eb3b3acde86cc464aea8f6bbbd35cb5395ed007 | ee033eba777f11e5abe9080027b2d0eb | active |
+| PROF_A_C_E | calico-host-01 |      docker     | a549f09ed2c9ae2a840edc834dea0d14e686a4bf6195c791db283aed7f8b5e48 | 011af79a778011e5abe9080027b2d0eb | active |
+| PROF_A_C_E | calico-host-02 |      docker     | ecbb19f9913236da5e03187fbfadf65a5761c49051adf54758d7f447852336fa | f6e1dd20777f11e5abe9080027b2d0eb | active |
+|   PROF_B   | calico-host-01 |      docker     | fb028123092ed9a91f279f1aa11517cbd5b023ec4daf382cce8471b3fe41e9af | f2497552777f11e5abe9080027b2d0eb | active |
+|   PROF_D   | calico-host-02 |      docker     | 6f103f1439c3d69ed014d4aa7f5b92dac7b892cbec627ac3b181ae4263e307de | faecf8be777f11e5abe9080027b2d0eb | active |
++------------+----------------+-----------------+------------------------------------------------------------------+----------------------------------+--------+
+
+```
+
+### calicoctl profile add <PROFILE>
+This command allows you to create security profiles to configure 
+networking policy for Calico endpoints. 
+
```

Comment:
"When you create a new profile with this command, it includes default rules and tags that make it act like a traditional security group: endpoints in the group can communicate freely.  However, you may modify the rules and tags to suit your needs.  See [AdvancedNetworkPolicy.md] for more discussion of profiles, rules and tags."